### PR TITLE
plugins/ocp: Add support for clear PCIe correctable error counters fe…

### PR DESCRIPTION
**Pull request by ikegami-t**
…ature

### CODE REVIEW

File: plugins/ocp/ocp-nvme.c [hunk 1]
```diff
@@ -942,3 +945,65 @@ static int eol_plp_failure_mode(int argc, char **argv, struct command *cmd,
...
+	OPT_ARGS(opts) = {
+		OPT_FLAG("no-uuid", 'n', &no_uuid,
+			 "Skip UUID index search (UUID index not required for OCP 1.0)"),
+		OPT_END()
+	};
+
+	err = parse_and_open(&dev, argc, argv, desc, opts);
+	if (err)
+		return err;
+	if (no_uuid == false) {
```
Comment (igaw) [inline plugins/ocp/ocp-nvme.c:970]: Personally I get easily confused by inverted logic. Wouldn't it make sense have a `bool uuid = true` as default?

File: plugins/ocp/ocp-nvme.c [hunk 2]
```diff
@@ -942,3 +945,65 @@ static int eol_plp_failure_mode(int argc, char **argv, struct command *cmd,
...
+		// OCP 2.0 requires UUID index support
```
Comment (igaw) [inline plugins/ocp/ocp-nvme.c:971]: `/* comment */`

**Review by igaw — COMMENTED** [review]
@arthurshau

**Comment by ikegami-t** [comment]
Thanks for your reviewing and comments. Just fixed the changes as mentioned.

**Pull request merged**